CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Nothing yet
 
 ## [4.0.0] - 2025-01-21
+
```

Comment:
Feels weird that this is empty, hmm. Should we include something here?

Reply:
Should we maybe repeat (some of?) the info from the GitHub release?

Reply:
Added some of the contents of the GitHub release.

I also added compare links for the individual alpha/beta releases (but before v4 alpha 21, we didn't publish GitHub releases so those aren't included)